src/main/java/audio/Sonifier.java

```diff
@@ -48,6 +47,13 @@ public static PlaybackController sonify(InstrumentDataRaw[] instrumentDataRaw, E
         double[][] outArrays = concatArrays(evInstrs, synthLines);
 
         double[] out = Mixer.mixAudioStreams(outArrays, evInstrs.length, synthLines.length - 1);
+
+        GlobalFxParam globalFxParam = new Harmonizer(globalFxParamRaw, lengthInBeats).gHarmonize();
```

Comment:
As far as I can see, the data for global and inst harmonization don't conflict inside Harmonizer so you could just use one object for both instead of creating a new one here.

Reply:
we are creating one Harmonizer for each instrument already,
I think it would be cleaner to create a new one for the globalFxData instead of using the last one twice, especially since we then could not do constructor overloading but had to create a new method addGlobalFx-Data, which theoretically could be called from every Harmonizer-Object, which seems kinda not very concern-separating